ruby/lib/rubocop/inclusive_code/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RuboCop
   module InclusiveCode
-    VERSION = '0.1.2'
+    VERSION = '0.1.4'
```

Comment:
0.1.3 was published after i updated the homepage, but i forgot to commit the change